# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
# This example is compatible with Dynamixel using protocol 1.0
joint1:
  ID: 1
  Return_Delay_Time: 0
  Operating_Mode: 5
